qlever.py

```diff
@@ -0,0 +1,488 @@
+#!/usr/bin/env python3
+
+# This is the QLever script (new version, written in Python).
+
+from configparser import ConfigParser, ExtendedInterpolation
+from datetime import datetime, date
+import os
+import glob
+import psutil
+import re
+import shlex
+import subprocess
+import sys
+import time
+import traceback
+
+BLUE = "\033[34m"
+RED = "\033[31m"
+BOLD = "\033[1m"
+NORMAL = "\033[0m"
+
+
+class ActionException(Exception):
+    pass
+
+
+class Actions:
+
+    def __init__(self):
+        self.config = ConfigParser(interpolation=ExtendedInterpolation())
+        self.config.read("Qleverfile.ini")
+        self.name = self.config['DEFAULT']['name']
```

Comment:
Do those give reasonable error messages when there is no, or an ill-formed Qleverfile?